engine/src/main/java/com/redhat/cloud/notifications/templates/ResourceOptimizationService.java

```diff
@@ -0,0 +1,40 @@
+package com.redhat.cloud.notifications.templates;
+
+import com.redhat.cloud.notifications.models.EmailSubscriptionType;
+import io.quarkus.qute.CheckedTemplate;
+import io.quarkus.qute.TemplateInstance;
+
+// Name needs to be "ResourceOptimizationService" to read templates from resources/templates/ResourceOptimizationService
+public class ResourceOptimizationService implements EmailTemplate {
```

Comment:
This class will need to be declared in `EmailTemplateFactory`. Here's an example:
https://github.com/RedHatInsights/notifications-backend/blob/a0bf02aaf261075b28cd3790cce2e073b123ecad/engine/src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateFactory.java#L49-L50